### ISSUE TITLE
Undecorated and resizeable Window not dispaly correct pointer icon

### ISSUE THREAD

**Issue by sonder-joker** [JetBrains/compose-jb#1075]
Environment: Windows
Version: 1.0.0-alpha3-build315
I don't know if this is expected behavior or need process it specially.
example:
```kotlin
fun main(args: Array<String>) = application {
    MiraiComposeWindow({

    }) {
        Text("test")
    }
}

@Composable
fun MiraiComposeWindow(
    onCloseRequest: () -> Unit,
    state: WindowState = rememberWindowState(size = MiraiComposeWindowSize),
    visible: Boolean = true,
    enabled: Boolean = true,
    focusable: Boolean = true,
    alwaysOnTop: Boolean = false,
    onPreviewKeyEvent: (KeyEvent) -> Boolean = { false },
    onKeyEvent: (KeyEvent) -> Boolean = { false },
    content: @Composable WindowScope.() -> Unit,
) = Window(
    onCloseRequest = onCloseRequest,
    state = state,
    visible = visible,
    icon = null,
    undecorated = true,
    resizable = true,
    enabled = enabled,
    focusable = focusable,
    alwaysOnTop = alwaysOnTop,
    onPreviewKeyEvent = onPreviewKeyEvent,
    onKeyEvent = onKeyEvent
) {
    WindowArea(
        minimizeButton = {
            Button(
                onClick = {
                    state.isMinimized = true
                }
            ) {
            }
        },
        maximizeButton = {
            Button(
                onClick = {
                    state.placement = when (state.placement) {
                        WindowPlacement.Floating -> WindowPlacement.Maximized
                        WindowPlacement.Maximized -> WindowPlacement.Floating
                        WindowPlacement.Fullscreen -> WindowPlacement.Floating
                    }
                }
            ) {
            }
        },
        exitButton = {
            Button(
                onClick = onCloseRequest
            ) {
            }
        },
        draggableAreaHeight = WindowDraggableHeight,
    ) {
        content()
    }
}

@Composable
private inline fun WindowScope.WindowArea(
    crossinline minimizeButton: @Composable () -> Unit = {},
    crossinline maximizeButton: @Composable () -> Unit = {},
    crossinline exitButton: @Composable () -> Unit = {},
    draggableAreaHeight: Dp = DialogDraggableHeight,
    content: @Composable () -> Unit
) {
    Column(Modifier.fillMaxSize()) {
        Row(
            modifier = Modifier.background(color = Color(75, 75, 75))
                .fillMaxWidth()
                .height(draggableAreaHeight)
                .padding(start = DraggableLeftStart, end = DraggableRightStart),
            verticalAlignment = Alignment.CenterVertically
        ) {
            WindowDraggableArea(
                modifier = Modifier.weight(1f)
            ) {
                Text(text = "", color = Color.White)
            }
            Row {
                minimizeButton()
                Spacer(modifier = Modifier.width(DraggableIconSpace))
                maximizeButton()
                Spacer(modifier = Modifier.width(DraggableIconSpace))
                exitButton()
            }
        }
        content()
    }
}

```

**Comment by igordmn** [comment]
It is not expected, cursor should change when we try to resize undecorated window.

The reason of this bug is this [issue](https://github.com/JetBrains/compose-jb/issues/997)

**Comment by prepor** [comment]
it should be fixed by https://github.com/JetBrains/androidx/pull/91

**Comment by igordmn** [comment]
> it should be fixed by JetBrains/androidx#91

Thanks!

There is a room for improvement. The proper behavior would be to maintain the cursor resizable during the dragging. Now we change the cursor to default. It can be easily achieved on Windows, but not on macOs.

1.0.0-rc2:

https://user-images.githubusercontent.com/5963351/143039596-104b3e28-0fc7-40cd-b966-26e81cec2a8a.mp4

Better to not close this issue, until we figure out, what we can do.

**Comment by akurasov** [comment]
Fixed in 1.0.0-rc2

Follow up issue #1444 is open

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5287">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
